mitosheet/src/jupyter/widget.tsx

```diff
@@ -60,13 +61,27 @@ export class ExampleView extends DOMWidgetView {
     }
 
     render(): void {
+        /**
+         * This is the final location where we use the shared variables. When we transition away from being
+         * a widget and to just a display() call, we will instead write these variables directly to the JS
+         * so that Mito can still be created with them.
+         * 
+         * We do this instead of making Mito creatable without these variables, because we assume that they 
+         * exist currently, and allowing them to be undefined is a big refactor.
```

Comment:
Another proposal is: just have a case at the top of Mito.tsx that checks if any of these are undefined and if they are it displays a message that says "hang tight while we build your super fast spreadsheet" or something like that. 

Not sure this is better.

Reply:
It's still requires a bunch of refactoring, b/c there are hooks that run off of the sheet data json, for example. So gonna leave for now.